spec/spec_helper.rb

```diff
@@ -1,55 +1,19 @@
-require 'simplecov'
-SimpleCov.start 'rails'
+# frozen_string_literal: true
 
 ENV["RAILS_ENV"] = "test"
 
 require File.expand_path("../dummy/config/environment.rb",  __FILE__)
 
-require 'ffaker'
-require 'rspec/rails'
-require 'database_cleaner'
-require 'capybara/rspec'
-require 'capybara/rails'
-require 'capybara/poltergeist'
-Capybara.register_driver(:poltergeist) do |app|
-  Capybara::Poltergeist::Driver.new app, timeout: 90
-end
-Capybara.javascript_driver = :poltergeist
-Capybara.default_max_wait_time = 10
-
-Dir[Rails.root.join("spec/support/**/*.rb")].each {|f| require f}
+require "rspec/rails"
 
-require 'spree/testing_support/factories'
-require 'spree/testing_support/url_helpers'
-require 'spree/testing_support/controller_requests'
-require 'spree/testing_support/authorization_helpers'
-require 'spree/testing_support/capybara_ext'
+Dir[File.join(File.dirname(__FILE__), "/support/**/*.rb")].each { |file| require file }
 
-FactoryBot.find_definitions
+require "spree_static_content/factories"
+require "spree/testing_support/controller_requests"
+require "solidus_support/extension/feature_helper"
 
 RSpec.configure do |config|
-  config.mock_with :rspec
-  config.use_transactional_fixtures = false
-
   config.include Spree::TestingSupport::ControllerRequests, type: :controller
   config.include Spree::TestingSupport::UrlHelpers
```

Comment:
This is already included [here](https://github.com/solidusio/solidus_support/blob/c552a2aa80f60c4964857d9b6401aede84e6ee6a/lib/solidus_support/extension/rails_helper.rb#L24), right? Or does it need to be readded for some reason?

Reply:
right, I removed the unnecessary lines there